src/acquisition/flusurv/flusurv_update.py

```diff
@@ -55,6 +70,23 @@
 rate_age_5: hospitalization rate for ages 65-74
 rate_age_6: hospitalization rate for ages 75-84
 rate_age_7: hospitalization rate for ages 85+
+rate_age_18t29: hospitalization rate for ages 18 to 29
```

Comment:
these column descriptions are good information...  is it worth duplicating in (or maybe moving to) the `.md` documentation file?